src/Components/Components/src/Forms/InputComponents/InputNumber.cs

```diff
@@ -62,19 +40,19 @@ static InputNumber()
         protected override void BuildRenderTree(RenderTreeBuilder builder)
         {
             builder.OpenElement(0, "input");
-            builder.AddAttribute(1, "step", _stepAttributeValue); // Before the splat so the user can override
+            builder.AddAttribute(1, "step", _stepAttributeValue);
             builder.AddMultipleAttributes(2, AdditionalAttributes);
             builder.AddAttribute(3, "type", "number");
             builder.AddAttribute(4, "class", CssClass);
-            builder.AddAttribute(5, "value", BindMethods.GetValue(CurrentValueAsString));
+            builder.AddAttribute(5, "value", BindConverter.FormatValue(CurrentValueAsString));
```

Comment:
Are you calling `FormatValue` just for consistency, or is it required? Since we know `CurrentValueAsString` is definitely a `string`, isn't this equivalent to:

```cs
builder.AddAttribute(5, "value", CurrentValueAsString);
```

Reply:
Consistency really. It's nice to say "this is what bind does" and point people at it.